native/src/components/RemoteContent.tsx

```diff
@@ -91,26 +90,30 @@ const RemoteContent = (props: PropType): ReactElement => {
   )
 
   return (
-    <StyledView onLayout={onLayout}>
-      <WebView
-        source={createHtmlSource(renderHtml(content, cacheDirectory, theme, language), resourceCacheUrl)}
-        originWhitelist={['*']} // Needed by iOS to load the initial html
-        javaScriptEnabled
-        dataDetectorTypes='none'
-        userAgent={userAgent}
-        domStorageEnabled={false}
-        showsVerticalScrollIndicator={false}
-        showsHorizontalScrollIndicator={false}
-        scrollEnabled={false} // to disable scrolling in iOS
-        onMessage={onMessage}
-        renderError={renderWebviewError}
-        onShouldStartLoadWithRequest={onShouldStartLoadWithRequest}
-        style={{
-          height: webViewHeight,
-          width: webViewWidth
-        }}
-      />
-    </StyledView>
+    <>
+      {!!content.length && (
```

Comment:
I fixed a unneeded rerendering when content.length === 0 --> was triggering the useEffect above

Reply:
I'd really favor moving the condition out of the return and using `content.length === 0` as check.
```suggestion
  if (content.length === 0) {
    return null
  }
```

Reply:
ìf (content.length)`
should be sufficient